style.css

```diff
@@ -113,11 +118,206 @@ h2 {
+.card-container { background: #e5e5e5; }
+
+.bg {
+  background-color: #fff;
+  padding: 5%;
+  border: 2px solid #dddbdb;
+  border-radius: 20px;
+}
+
+.container {
+  padding: 6%;
+  font-family: 'Poppins', sans-serif;
+  max-width: 768px;
+  margin: auto;
+  background: #e5e5e5;
+  display: grid;
+  grid-template-columns: 1fr;
+  font-weight: 700;
+  margin-bottom: 4rem;
+}
+
+.left-block { padding: 6%; }
+
+.primary-text {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  position: relative;
+  align-items: flex-start;
+}
+
+.tonic {
+  font-size: 30px;
+  font-weight: bolder;
+  font-family: 'poppins', sans-serif;
+  transform: translateX(-11%);
+  color: #344563;
+}
+
+.frame {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  color: #7a869a;
+}
+
+.frame ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 4rem;
+  width: 100%;
+}
+
+.frame ul li {
+  margin-left: 0.5rem;
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.frame .spans {
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  background: #c1c7d0;
+}
+
+.techs {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  width: 100%;
+  position: relative;
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  padding: 0.3rem;
+}
+
+.img {
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+}
+
+.techs ul {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+}
+
+.techs ul li {
+  margin-right: 1rem;
+  margin-bottom: 2rem;
+  width: 100%;
+  background-color: #e5e5e5;
+  color: #6070ff;
+  padding: 0.8rem 0.8rem;
+  border-radius: 15px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+button {
+  padding: 0.7rem 0.9rem;
+  cursor: pointer;
+  color: #6070ff;
+  font-size: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: bold;
+  background: transparent;
+  border-radius: 15px;
+  border: #6070ff solid 1px;
+  margin-bottom: 1rem;
+  transition: button 3s;
+}
+
+/* About styling */
+
+.container-b {
+  font-family: 'Poppins', sans-serif;
+  max-width: 768px;
+  margin: auto;
+  font-weight: 700;
+  border-radius: 0 20px 0 0;
+}
+
+.bg-b {
+  background-color: #fff;
+  padding: 5%;
+  border-radius: 0 60px 0 0;
+}
+
+.about-h1 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: bolder;
+  font-size: 45px;
+  margin-bottom: 1rem;
+}
+
+.about-p { margin-top: 1.5rem; }
+
+.connect { color: #6070ff; }
+
+.btn {
+  padding: 0.7rem 0.9rem;
+  cursor: pointer;
+  color: #6070ff;
+  font-size: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: bold;
+  background: transparent;
+  border-radius: 15px;
+  border: #6070ff solid 1px;
+  margin: 4rem 0;
+  transition: btn 3s;
+}
+
+button:hover,
+.btn:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
```

Comment:
- [ ] Great job adding the hover state on your buttons. Kindly add the pressed state as suggested by the previous reviewer. You can use the `:active` pseudoclass for the pressed state.
![image](https://github.com/Isaac-ej/Portfolio_Website/assets/94551484/b114cd19-7155-49ed-b45a-116a7823c86b)

Reply:
- [ ] Please implement this change as both previous reviewers requested 🙏🏽 
You can achieve that this way 👇🏽 

```sh
button:active {
background-color: #2230D2;
color: #fff;
}
```